server/form-pages/apply/risk-and-need-factors/access-and-healthcare/accessNeedsFurtherQuestions.test.ts

```diff
@@ -0,0 +1,88 @@
+import { itShouldHaveNextValue, itShouldHavePreviousValue } from '../../../shared-examples'
+
+import AccessNeedsFurtherQuestions, { AccessNeedsFurtherQuestionsBody } from './accessNeedsFurtherQuestions'
+
+import { applicationFactory, personFactory } from '../../../../testutils/factories'
+import { DateFormats } from '../../../../utils/dateUtils'
+
+jest.mock('../../../../utils/retrieveQuestionResponseFromApplicationOrAssessment', () => {
+  return {
+    retrieveOptionalQuestionResponseFromApplicationOrAssessment: jest.fn(() => ['pregnancy']),
+  }
+})
+
+describe('AccessNeedsFurtherQuestions', () => {
+  const person = personFactory.build({ name: 'John Wayne' })
+  const application = applicationFactory.build({ person })
+
+  const expectedDeliveryDate = new Date(2023, 1, 19)
+  const body: AccessNeedsFurtherQuestionsBody = {
+    needsWheelchair: 'yes',
+    mobilityNeeds: 'mobility needs',
+    visualImpairment: 'visual impairment',
+    expectedDeliveryDate: DateFormats.dateObjToIsoDate(expectedDeliveryDate),
+    ...DateFormats.dateObjectToDateInputs(expectedDeliveryDate, 'expectedDeliveryDate'),
+    otherPregnancyConsiderations: 'none',
+    childRemoved: 'no',
+    isPersonPregnant: 'yes',
+  }
+
+  describe('title', () => {
+    expect(new AccessNeedsFurtherQuestions({}, application).title).toBe('Access, cultural and healthcare needs')
+  })
+
+  describe('body', () => {
+    it('should set the body', () => {
+      const page = new AccessNeedsFurtherQuestions(body, application)
+      expect(page.body).toEqual({
+        needsWheelchair: 'yes',
+        visualImpairment: 'visual impairment',
+        mobilityNeeds: 'mobility needs',
+        isPersonPregnant: 'yes',
+        expectedDeliveryDate: DateFormats.dateObjToIsoDate(expectedDeliveryDate),
+        'expectedDeliveryDate-year': '2023',
+        'expectedDeliveryDate-month': '2',
+        'expectedDeliveryDate-day': '19',
+        otherPregnancyConsiderations: 'none',
+        childRemoved: 'no',
+      })
+    })
+  })
+
+  itShouldHaveNextValue(new AccessNeedsFurtherQuestions({}, application), 'covid')
+  itShouldHavePreviousValue(new AccessNeedsFurtherQuestions({}, application), 'access-needs')
+
+  describe('errors', () => {
+    it('should return errors if there are no responses to needsWheelchair or isPersonPregnant questions', () => {
+      const page = new AccessNeedsFurtherQuestions({}, application)
+      expect(page.errors()).toEqual({
+        needsWheelchair: 'You must confirm the need for a wheelchair',
+        isPersonPregnant: 'You must confirm if the person is pregnant',
+      })
+    })
+    it('if isPersonIsPregnant is yes, should return errors if there are no responses to expectedDeliveryDate or childRemoved questions', () => {
```

Comment:
Minor thing, but I tend to prefer test names to follow on from `it...` - for me:

```
it('should return errors if there are no responses to expectedDeliveryDate or childRemoved question and isPersonPregnant is yes')
```

Reads better.

Also, there's a missing linebreak between the two tests